eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -288,11 +322,10 @@ epochTransition = do
     removedProps =
       foldl' lookupAction mempty $
         Seq.fromList (Set.toList rsRemoved)
-    lState = returnProposalDeposits removedProps $ esLState es'
+    lState = returnProposalDeposits removedProps $ esLState epochState'
     es'' =
-      es'
-        { esAccountState = acnt'
```

Comment:
`acnt'` is already part of the `EpochState` a few lines above. And after that we update it in `applyEnactedWithdrawals` and so, it need not be updated here again.